server/engine/src/main/antlr4/org/eclipse/lsp/cobol/core/CompilerDirectivesParser.g4

```diff
@@ -224,6 +225,14 @@ cobolCompilerOption
    | ZWB | NOZWB | deprecatedCompilerOptions
    ;
 
+cobolJavaInteroperabilityOptions:
+            ((JAVAIOP | JIOP) LPARENCHAR (
+                (OUTPATH | OP) LPARENCHAR LITERAL RPARENCHAR
+                | (NOJVMINITOPTIONS | NOJVMI)
+                | (JVMINITOPTIONS | JVMI) LPARENCHAR LITERAL RPARENCHAR
+                | (NOJAVA64 | JAVA64)
+            ) RPARENCHAR | (NOJAVAIOP | NOJIOP)) extraneousInput?;
```

Comment:
Why is the `extraneousInput` in this grammar rule? It has the ability to silently suppress all kinds of errors.